packages/venia-ui/lib/components/CheckoutPage/ShippingMethod/shippingMethod.js

```diff
@@ -39,82 +38,78 @@ const ShippingMethod = props => {
-
     const classes = mergeClasses(defaultClasses, props.classes);
 
     let contents;
-    switch (displayState) {
-        case displayStates.EDITING:
-            {
-                const continueDisabled =
-                    pageIsUpdating || !shippingMethods.length;
+    if (displayState === displayStates.EDITING) {
+        const lowestCostShippingMethod = {
+            shipping_method: serializeShippingMethod(shippingMethods[0])
+        };
+        const isContinueDisabled = pageIsUpdating || !shippingMethods.length;
 
-                contents = (
-                    <div className={classes.root}>
-                        <h3 className={classes.editingHeading}>
-                            {'Shipping Method'}
-                        </h3>
-                        <Form className={classes.form} onSubmit={handleSubmit}>
-                            <ShippingRadios
-                                isLoading={isLoading}
-                                selectedShippingMethod={selectedShippingMethod}
-                                shippingMethods={shippingMethods}
-                            />
-                            {!isLoading && (
-                                <div className={classes.formButtons}>
-                                    <Button
-                                        priority="normal"
-                                        type="submit"
-                                        disabled={continueDisabled}
-                                    >
-                                        {'Continue to Payment Information'}
-                                    </Button>
-                                </div>
-                            )}
-                        </Form>
-                    </div>
-                );
-            }
-            break;
+        // The final JSX for the edit view.
+        contents = (
+            <div className={classes.root}>
+                <h3 className={classes.editingHeading}>{'Shipping Method'}</h3>
+                <Form
+                    className={classes.form}
+                    initialValues={lowestCostShippingMethod}
+                    onSubmit={handleSubmit}
+                >
+                    <ShippingRadios
+                        isLoading={isLoading}
+                        shippingMethods={shippingMethods}
+                    />
+                    {!isLoading && (
+                        <div className={classes.formButtons}>
+                            <Button
+                                priority="normal"
+                                type="submit"
+                                disabled={isContinueDisabled}
+                            >
+                                {'Continue to Payment Information'}
+                            </Button>
+                        </div>
+                    )}
+                </Form>
+            </div>
+        );
+    } else {
+        const updateFormInitialValues = {
```

Comment:
How about getting this from the talon?

Reply:
Building on your previous suggestions, we can actually just use `selectedShippingMethod.serializedValue` here and achieve not having to import the `serialize` function here 🥇 

I did want to keep the definition of this object in the component though since I considered it more of a UI concern.

Reply:
Resolved in https://github.com/magento/pwa-studio/pull/2382/commits/d43ed986d954ba5e9ce3b1fa07046e9d1fdf0df6.